pysatSpaceWeather/instruments/methods/ace.py

```diff
@@ -15,6 +15,9 @@
 import pysat
 
 logger = pysat.logger
+clean_warn = {'dusty': [('logger', 'WARN',
+                         "unused clean level 'dusty', reverting to 'clean'",
+                         'clean')]}
```

Comment:
Noting that in pysatNASA 0.0.5, `clean_warn` is used as a function in the methods files to warn users that no clean routine is available. We don't necessarily need to be consistent across packages, but wanted to bring this up.

https://github.com/pysat/pysatNASA/blob/e62b8c64b4701b9c41a082776c46bd140d18655e/pysatNASA/instruments/methods/general.py#L75-L89

Reply:
I chose this to be consistent with the `_clean_warn` attribute, didn't realize pysatNASA had that function name 🐞 But yeah, we don't need to be that consistent across packages.